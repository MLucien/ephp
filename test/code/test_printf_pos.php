<?php
$num = 5;
$ubicación = 'árbol';

$formato = 'El %2$s contiene %1$d monos.' . "\n" .
           'Es un bonito %2$s con %1$d monos.' . "\n";
printf($formato, $num, $ubicación);
