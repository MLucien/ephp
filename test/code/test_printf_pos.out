El árbol contiene 5 monos.
Es un bonito árbol con 5 monos.
